akka-http/src/main/scala/akka/http/scaladsl/server/directives/HeaderDirectives.scala

```diff
@@ -68,6 +69,7 @@ trait HeaderDirectives {
     headerValue(f).map(Some(_): Option[T]).recoverPF {
       case Nil ⇒ provide(None)
     }
+  //#
```

Comment:
AFAIR includecode2 can work with `#optionalHeaderValue` for those. But maybe OK to include with docs in any case :)